### PR TITLE
Document packaging process

### PR DESCRIPTION
**Pull request by edsantiago**
High-level overview of packaging, intended for Linux distros.

Closes: RUN-1356

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
Document the process for packaging Podman for a new Linux distro
```

### CODE REVIEW

**Comment by edsantiago** [comment]
Thank you! I meant to mark this `draft` yesterday, have just done so. My goal was to get preliminary feedback. It sounds like this is the desired approach, so I will continue polishing it up.

File: DISTRO_PACKAGE.md [hunk 1]
```diff
@@ -1,101 +1,98 @@
...
+[netavark](https://github.com/containers/netavark/blob/main/rpm/netavark.spec),
+[aardvark-dns](https://github.com/containers/aardvark-dns/blob/main/rpm/aardvark-dns.spec).
 
-- containernetworking-plugins
-```
-Provides: container-network-stack = 1
-```
+Netavark and aardvark-dns must be packaged in lockstep down
+to the major-minor level: version `X.Y` of either will _only_
+work with `X.Y` of the other. If you are responsible for
```
[the remark this answers is not in the document]
Comment (edsantiago) [reply DISTRO_PACKAGE.md]: Changed to "guaranteed to work".

File: DISTRO_PACKAGE.md [hunk 2]
```diff
@@ -1,101 +1,98 @@
...
+## Metapackage: containers-common
 
-## Listing bundled dependencies
-If you need to list the bundled dependencies in your packaging sources, you can
-process the `go.mod` file in the upstream source.
-For example, Fedora's packaging source uses:
+This package provides config files, man pages, and (at the
+packaging level) dependencies. You will probably be responsible
+for packaging this. (You could choose to include all of this
+as part of the Podman package, but that would be a mistake.)
```
[the remark this answers is not in the document]
Comment (edsantiago) [reply DISTRO_PACKAGE.md]: Expanded. Thanks.

File: DISTRO_PACKAGE.md [hunk 3]
```diff
@@ -1,101 +1,98 @@
...
 
-On Fedora 36 and newer, fresh installations of Podman v4 will
-automatically install Aardvark-dns along with Netavark.
+This document assumes you are able to build executables up to and
+including `make install`.
+See [Building from Source](https://podman.io/docs/installation#building-from-source)
+on podman.io for possibly-outdated instructions.
 
-To install Podman v4:
+---------------------
```
[the remark this answers is not in the document]
Comment (edsantiago) [reply DISTRO_PACKAGE.md]: My eye finds them helpful when navigating a page, but not a big deal. Removed.

**Comment by edsantiago** [comment]
@containers/podman-maintainers this is ready for review

**Review by Luap99 — APPROVED** [review]
LGTM

**Comment by lsm5** [comment]
@edsantiago thanks a lot! LGTM :+1:

**Pull request merged**